streamly-process.cabal

```diff
@@ -60,7 +60,7 @@ library
   build-depends:
       base              >= 4.8   && < 5
     , process           >= 1.0   && < 1.7
-    , streamly          >= 0.7.1 && < 0.8
+    , streamly          >= 0.8   && < 0.9
```

Comment:
We are depending on the internal imports. Do you want to make the upper bound inclusive of the latest working minor release? 0.8.0 in this case.